### ISSUE TITLE
Security warnings didn't show

### ISSUE THREAD

**Issue by serrq** [Droid-ify/client#198]
As users' words and F-droid’s actions demonstrate, vulnerabilities have been discovered in the Librera code, which is why F-droid reports a «warning» on the app page.

https://f-droid.org/en/packages/com.foobnix.pro.pdf.reader/

However, I find out about this vulnerability only by randomly reading a post on Reddit.

https://www.reddit.com/r/fossdroid/comments/zq6c2i/librera_got_flagged_on_fdroid_known_vulnerability/

Can we make persistent notifications warnings in the notification bar?

**Comment by Iamlooker** [comment]
I don't think this would be very feasible, because we get the whole app list everytime so the already available apps are updates to show new data. I can try checking if a new 'Anti-Feature' is added to an installed app... I will teat and reply back asap
